bin/test.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2015 Google. All rights reserved. Use of this source code is
+// governed by a BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:grinder/grinder.dart';
+
+/**
+ * A small command-line utility to leverage the grinder APIs to make it easy to
+ * run CLI or web app tests.
+ */
+void main(List<String> args) {
+  if (args.length != 1) {
+    print(
+        'usage: pub global activate grinder:test <filepath>');
```

Comment:
Do you mean `usage: pub global run grinder:test <filepath>`?  (using `run` instead of  `activate`?)

> pub global activate grinder:test
Resolving dependencies... (1.0s)
Could not find package grinder:test at https://pub.dartlang.org.
Depended on by:
- pub global activate